plugin/forward/forward.go

```diff
@@ -186,10 +186,12 @@ func (f *Forward) ForceTCP() bool { return f.forceTCP }
 func (f *Forward) List() []*Proxy { return f.p.List(f.proxies) }
 
 var (
-	errInvalidDomain = errors.New("invalid domain for forward")
-	errNoHealthy     = errors.New("no healthy proxies")
```

Comment:
You are removing 4 and adding only 3.
I guess the errInvalidDomain is not used ...

Reply:
Yes. It is not used, so removed.